src/delta_lake/delta_service.py

```diff
@@ -248,6 +266,21 @@ def sample_delta_table(
     Returns:
         A list of dictionaries, where each dictionary represents a row.
     """
+    namespace = "sample"
+    params = {
+        "database": database,
+        "table": table,
+        "limit": limit,
+        "columns": columns,
```

Comment:
Should these be sorted to avoid different cache keys for the same contents?

Reply:
yea. those are sorted in the `_generate_cache_key` function.

Reply:
That'll only sort the keys, not the arrays in the keys, unless I'm missing something

Reply:
yup. I see. updated.